src/pages/settings/Preferences/PreferencesPage.js

```diff
@@ -0,0 +1,116 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import ONYXKEYS from '../../../ONYXKEYS';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+import CONST from '../../../CONST';
+import * as User from '../../../libs/actions/User';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import Switch from '../../../components/Switch';
+import withLocalize, {withLocalizePropTypes, localeProviderPropTypes} from '../../../components/withLocalize';
+import compose from '../../../libs/compose';
+import withEnvironment, {environmentPropTypes} from '../../../components/withEnvironment';
+import TestToolMenu from '../../../components/TestToolMenu';
+import MenuItemWithTopDescription from '../../../components/MenuItemWithTopDescription';
+
+const propTypes = {
+    /** The chat priority mode */
+    priorityMode: PropTypes.string,
+
+    /** The details about the user that is signed in */
+    user: PropTypes.shape({
+        /** Whether or not the user is subscribed to news updates */
+        isSubscribedToNewsletter: PropTypes.bool,
+    }),
+
+    ...withLocalizePropTypes,
+    ...localeProviderPropTypes,
+    ...environmentPropTypes,
+};
+
+const defaultProps = {
+    priorityMode: CONST.PRIORITY_MODE.DEFAULT,
+    user: {},
+};
+
+const PreferencesPage = (props) => {
+    const priorityModes = props.translate('priorityModePage.priorityModes');
+    const languages = props.translate('languagePage.languages');
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <HeaderWithCloseButton
+                title={props.translate('common.preferences')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <ScrollView style={[styles.flex1, styles.mt3]}>
+                <View style={styles.mb6}>
+                    <Text style={[styles.textLabelSupporting, styles.mb2, styles.ml8, styles.mr8]} numberOfLines={1}>
+                        {props.translate('common.notifications')}
+                    </Text>
+                    <View style={[styles.flexRow, styles.mb4, styles.justifyContentBetween, styles.ml8, styles.mr8]}>
```

Comment:
NAB - I think it would make sense to add a new `mh8` style that combines these

```suggestion
                    <View style={[styles.flexRow, styles.mb4, styles.justifyContentBetween, styles.mh8]}>
```

Reply:
fixed.